PathOfBuildingCommunity/PathOfBuilding
# Ensure item base type follows item title for unique items

Fixes data inconsistency.

### Description of the problem being solved:
Some items have Variant information between the item title and the item type (or base_name as the code has it)

### Steps taken to verify a working solution:
- Open PoB
- Search for Story of the Vaal unique item.
- Double click on it
- Check that the variant dropdown still works
![image](https://user-images.githubusercontent.com/4302241/207025636-f53c4dcf-2952-43e1-8818-dc37c1f1bd16.png)
![image](https://user-images.githubusercontent.com/4302241/207025704-39b74dba-cce6-4f40-b8bc-0e1b8e8371a0.png)

### Link to a build that showcases this PR:

### Before screenshot:

### After screenshot:
